examples/osxpkg/test_install.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ex
+
+# $2 is the install location, ($HOME by default)
+if [ xxx$PREFIX == 'xxx' ]; then
+    PREFIX=$(cd "$2/__NAME_LOWER__"; pwd)
+fi
+
+echo '## Hello from Post_install script ' > $HOME/postinstall.txt
```

Comment:
I looked for "hello from" in https://github.com/conda/constructor/runs/7542349027?check_suite_focus=true but I don't see it. Am I missing it somewhere else? I expected this to run and have some visible output in an OSX CI run

Reply:
I don't think the output shows in the CI logs, only in the GUI installer (and system logs, but guess what, when I looked into this, the system logs contain details of ALL VMs running on the machine, which makes it quite noisy :D)

Example output from the `ExtraFiles` example (look for `test -f`):

```
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: @(#)PROGRAM:Install  PROJECT:Install-1000
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: @(#)PROGRAM:Installer  PROJECT:Installer-1018
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Hardware: MacBookPro17,1 @ 0 MHz (x 8), 16384 MB RAM
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Running OS Build: macOS 11.2.3 (20D91)
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: USER=jrodriguez
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: __CFBundleIdentifier=com.apple.installer
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: COMMAND_MODE=unix2003
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: DISPLAY=/private/tmp/com.apple.launchd.qfveRuDBeT/org.xquartz:0
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: LOGNAME=jrodriguez
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: PATH=/usr/bin:/bin:/usr/sbin:/sbin
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.cn1P6Dop2l/Listeners
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: SHELL=/bin/zsh
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: HOME=/Users/jrodriguez
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x0
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: TMPDIR=/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/T/
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: XPC_SERVICE_NAME=application.com.apple.installer.1152921500312163448.1152921500312163453
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Env: XPC_FLAGS=0x0
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: ExtraFiles  Installation Log
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Opened from: /Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: No localized entry for No license
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Package Authoring Error: <background> has an unsupported MIME type: image/data
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Package Authoring Error: <background-darkAqua> has an unsupported MIME type: image/data
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Package Authoring Error: <background_scaling> has an unsupported MIME type: X-NSObject/NSNumber
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Package Authoring Error: <background_alignment> has an unsupported MIME type: X-NSObject/NSNumber
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Package Authoring Error: <layout-direction> has an unsupported MIME type: X-NSObject/NSNumber
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Product archive /Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg trustLevel=100
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: External component packages (4) trustLevel=100 (trust evaluation failed: Error Domain=PKInstallErrorDomain Code=101 "The package “ExtraFiles-X-MacOSX-arm64.pkg” is not signed." UserInfo={NSURL=#cacheclean.pkg -- file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#Distribution, PKInstallPackageIdentifier=io.continuum.pkg.cacheclean, NSLocalizedDescription=The package “ExtraFiles-X-MacOSX-arm64.pkg” is not signed.})
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: No localized entry for No license
Jul 27 20:33:58 Jaimes-MBP Installer[6147]: Could not load resource license: (null)
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: ================================================================================
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: User picked Standard Install
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Choices selected for installation:
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 	Install: ""
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 	Install: "Install ExtraFiles"
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 		ExtraFiles-X-MacOSX-arm64.pkg#main.pkg : io.continuum.pkg.main : X
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 		ExtraFiles-X-MacOSX-arm64.pkg#postinstall.pkg : io.continuum.pkg.postinstall : 0
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 	Install: "Clear the package cache"
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: 		ExtraFiles-X-MacOSX-arm64.pkg#cacheclean.pkg : io.continuum.pkg.cacheclean : 0
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: ================================================================================
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: It took 0.00 seconds to summarize the package selections.
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: location = file://localhost
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: file://localhost/Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#main.pkg
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: file://localhost/Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#postinstall.pkg
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: -[IFDInstallController(Private) _buildInstallPlanReturningError:]: file://localhost/Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#cacheclean.pkg
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Set authorization level to none for session
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Will use PK session
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Using authorization level of none for IFPKInstallElement
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: PackageKit: Bypassing the enforcement checker since the destination is read/write
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Starting installation:
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Configuring volume "Macintosh HD"
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Preparing disk for local booted install.
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Free space on "Macintosh HD": 680,7 GB (680696520704 bytes).
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Create temporary directory "/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/T//Install.6147K5b4yo"
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: IFPKInstallElement (3 packages)
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: Current Path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Adding client PKInstallDaemonClient pid=6147, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Jul 27 20:34:04 Jaimes-MBP Installer[6147]: PackageKit: Enqueuing install with framework-specified quality of service (utility)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: ----- Begin install -----
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: request=PKInstallRequest <3 packages, destination=/Users/jrodriguez>
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: packages=(
	    "PKLeopardPackage <id=io.continuum.pkg.main, version=X, url=file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#main.pkg>",
	    "PKLeopardPackage <id=io.continuum.pkg.postinstall, version=0, url=file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#postinstall.pkg>",
	    "PKLeopardPackage <id=io.continuum.pkg.cacheclean, version=0, url=file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#cacheclean.pkg>"
	)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Set reponsibility for install to 6147
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Will do receipt-based obsoleting for package identifier io.continuum.pkg.main (prefix path=/)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Will do receipt-based obsoleting for package identifier io.continuum.pkg.postinstall (prefix path=/)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Will do receipt-based obsoleting for package identifier io.continuum.pkg.cacheclean (prefix path=/)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Extracting file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#main.pkg (destination=/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/C/PKInstallSandboxManager/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.activeSandbox/Root, uid=501)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Extracting file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#postinstall.pkg (destination=/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/C/PKInstallSandboxManager/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.activeSandbox/Root, uid=501)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: Extracting file:///Users/jrodriguez/devel/constructor/ExtraFiles-X-MacOSX-arm64.pkg#cacheclean.pkg (destination=/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/C/PKInstallSandboxManager/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.activeSandbox/Root, uid=501)
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: prevent user idle system sleep
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit: suspending backupd
Jul 27 20:34:04 Jaimes-MBP installd[96293]: PackageKit (package_script_service): Preparing to execute script "./preinstall" in /private/tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.main.zZaMmd
Jul 27 20:34:04 Jaimes-MBP package_script_service[96294]: PackageKit: Executing script "preinstall" in /tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.main.zZaMmd
Jul 27 20:34:04 Jaimes-MBP package_script_service[96294]: Set responsibility to pid: 6147, responsible_path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
Jul 27 20:34:05 Jaimes-MBP package_script_service[96294]: Responsibility set back to self.
Jul 27 20:34:05 Jaimes-MBP installd[96293]: PackageKit: Using trashcan path /var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/T/PKInstallSandboxTrash/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.sandboxTrash for sandbox /var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/C/PKInstallSandboxManager/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.activeSandbox
Jul 27 20:34:05 Jaimes-MBP installd[96293]: PackageKit: Shoving /var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/C/PKInstallSandboxManager/A3593BA2-BD5D-4C06-ABBA-E2D44AF833DE.activeSandbox/Root (1 items) to /Users/jrodriguez
Jul 27 20:34:05 Jaimes-MBP installd[96293]: PackageKit (package_script_service): Preparing to execute script "./postinstall" in /private/tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.main.zZaMmd
Jul 27 20:34:05 Jaimes-MBP package_script_service[96294]: PackageKit: Executing script "postinstall" in /tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.main.zZaMmd
Jul 27 20:34:05 Jaimes-MBP package_script_service[96294]: Set responsibility to pid: 6147, responsible_path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
Jul 27 20:34:05 Jaimes-MBP package_script_service[96294]: ./postinstall: PREFIX=/Users/jrodriguez/extrafiles
Jul 27 20:34:28 Jaimes-MBP package_script_service[96294]: ./postinstall: 
	  0%|          | 0/17 [00:00<?, ?it/s]
	Extracting : pip-22.1.2-py310hca03da5_0.conda:   0%|          | 0/17 [00:00<?, ?it/s]
	Extracting : pip-22.1.2-py310hca03da5_0.conda:   6%|5         | 1/17 [00:00<00:14,  1.13it/s]
	Extracting : setuptools-61.2.0-py310hca03da5_0.conda:   6%|5         | 1/17 [00:00<00:14,  1.13it/s]
	Extracting : zlib-1.2.12-h5a0b063_2.conda:  12%|#1        | 2/17 [00:00<00:13,  1.13it/s]           
	Extracting : bzip2-1.0.8-h620ffc9_4.conda:  18%|#7        | 3/17 [00:00<00:12,  1.13it/s]
	Extracting : ca-certificates-2022.07.19-hca03da5_0.conda:  24%|##3       | 4/17 [00:00<00:11,  1.13it/s]
	Extracting : readline-8.1.2-h1a28f6b_1.conda:  29%|##9       | 5/17 [00:00<00:10,  1.13it/s]            
	Extracting : xz-5.2.5-h1a28f6b_1.conda:  35%|###5      | 6/17 [00:00<00:09,  1.13it/s]      
	Extracting : wheel-0.37.1-pyhd3eb1b0_0.conda:  41%|####1     | 7/17 [00:00<00:08,  1.13it/s]
	Extracting : libffi-3.4.2-hc377ac9_4.conda:  47%|####7     | 8/17 [00:00<00:07,  1.13it/s]  
	Extracting : libcxx-12.0.0-hf6beb65_1.conda:  53%|#####2    | 9/17 [00:00<00:07,  1.13it/s]
	Extracting : sqlite-3.38.5-h1058600_0.conda:  59%|#####8    | 10/17 [00:00<00:06,  1.13it/s]
	Extracting : certifi-2022.6.15-py310hca03da5_0.conda:  65%|######4   | 11/17 [00:00<00:05,  1.13it/s]
	Extracting : tk-8.6.12-hb8d0fd4_0.conda:  71%|#######   | 12/17 [00:00<00:04,  1.13it/s]             
	Extracting : ncurses-6.3-h1a28f6b_3.conda:  76%|#######6  | 13/17 [00:01<00:03,  1.13it/s]
	Extracting : ncurses-6.3-h1a28f6b_3.conda:  82%|########2 | 14/17 [00:01<00:00, 12.75it/s]
	Extracting : openssl-1.1.1q-h1a28f6b_0.conda:  82%|########2 | 14/17 [00:01<00:00, 12.75it/s]
	Extracting : tzdata-2022a-hda174b7_0.conda:  88%|########8 | 15/17 [00:01<00:00, 12.75it/s]  
	Extracting : python-3.10.4-hbdb9e5c_0.tar.bz2:  94%|#########4| 16/17 [00:02<00:00, 12.75it/s]
	Extracting : python-3.10.4-hbdb9e5c_0.tar.bz2: 100%|##########| 17/17 [00:02<00:00,  7.11it/s
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall: Solving environment: ...working... done
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall: ## Package Plan ##
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   environment location: /Users/jrodriguez/extrafiles
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   added / updated specs:
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - bzip2==1.0.8=h620ffc9_4
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - ca-certificates==2022.07.19=hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - certifi==2022.6.15=py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - libcxx==12.0.0=hf6beb65_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - libffi==3.4.2=hc377ac9_4
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - ncurses==6.3=h1a28f6b_3
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - openssl==1.1.1q=h1a28f6b_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - pip==22.1.2=py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - python==3.10.4=hbdb9e5c_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - readline==8.1.2=h1a28f6b_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - setuptools==61.2.0=py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - sqlite==3.38.5=h1058600_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - tk==8.6.12=hb8d0fd4_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - tzdata==2022a=hda174b7_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - wheel==0.37.1=pyhd3eb1b0_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - xz==5.2.5=h1a28f6b_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:     - zlib==1.2.12=h5a0b063_2
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall: The following NEW packages will be INSTALLED:
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   bzip2              pkgs/main/osx-arm64::bzip2-1.0.8-h620ffc9_4
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   ca-certificates    pkgs/main/osx-arm64::ca-certificates-2022.07.19-hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   certifi            pkgs/main/osx-arm64::certifi-2022.6.15-py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   libcxx             pkgs/main/osx-arm64::libcxx-12.0.0-hf6beb65_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   libffi             pkgs/main/osx-arm64::libffi-3.4.2-hc377ac9_4
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   ncurses            pkgs/main/osx-arm64::ncurses-6.3-h1a28f6b_3
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   openssl            pkgs/main/osx-arm64::openssl-1.1.1q-h1a28f6b_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   pip                pkgs/main/osx-arm64::pip-22.1.2-py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   python             pkgs/main/osx-arm64::python-3.10.4-hbdb9e5c_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   readline           pkgs/main/osx-arm64::readline-8.1.2-h1a28f6b_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   setuptools         pkgs/main/osx-arm64::setuptools-61.2.0-py310hca03da5_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   sqlite             pkgs/main/osx-arm64::sqlite-3.38.5-h1058600_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   tk                 pkgs/main/osx-arm64::tk-8.6.12-hb8d0fd4_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   tzdata             pkgs/main/noarch::tzdata-2022a-hda174b7_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   wheel              pkgs/main/noarch::wheel-0.37.1-pyhd3eb1b0_0
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   xz                 pkgs/main/osx-arm64::xz-5.2.5-h1a28f6b_1
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall:   zlib               pkgs/main/osx-arm64::zlib-1.2.12-h5a0b063_2
Jul 27 20:34:29 Jaimes-MBP package_script_service[96294]: ./postinstall: Preparing transaction: ...working... done
Jul 27 20:34:35 Jaimes-MBP package_script_service[96294]: ./postinstall: Executing transaction: ...working... done
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: Python 3.10.4
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: installation to /Users/jrodriguez/extrafiles finished.
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: Responsibility set back to self.
Jul 27 20:34:36 Jaimes-MBP installd[96293]: PackageKit (package_script_service): Preparing to execute script "./postinstall" in /private/tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.postinstall.3RXU8c
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: PackageKit: Executing script "postinstall" in /tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.postinstall.3RXU8c
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: Set responsibility to pid: 6147, responsible_path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: + '[' xxx = xxx ']'
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ echo /Users/jrodriguez/extrafiles
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ sed -e s,//,/,g
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: + PREFIX=/Users/jrodriguez/extrafiles
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ cd /Users/jrodriguez/extrafiles
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ pwd
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: + PREFIX=/Users/jrodriguez/extrafiles
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: + test -f /Users/jrodriguez/extrafiles/more_data/README.md
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: ./postinstall: + test -f /Users/jrodriguez/extrafiles/something2.txt
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: Responsibility set back to self.
Jul 27 20:34:36 Jaimes-MBP installd[96293]: PackageKit (package_script_service): Preparing to execute script "./postinstall" in /private/tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.cacheclean.47skYG
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: PackageKit: Executing script "postinstall" in /tmp/PKInstallSandbox.fHDBTe/Scripts/io.continuum.pkg.cacheclean.47skYG
Jul 27 20:34:36 Jaimes-MBP package_script_service[96294]: Set responsibility to pid: 6147, responsible_path: /System/Library/CoreServices/Installer.app/Contents/MacOS/Installer
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: ./postinstall: + PREFIX=/Users/jrodriguez/extrafiles
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ cd /Users/jrodriguez/extrafiles
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: ./postinstall: ++ pwd
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: ./postinstall: + PREFIX=/Users/jrodriguez/extrafiles
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: ./postinstall: + rm -rf /Users/jrodriguez/extrafiles/pkgs
Jul 27 20:34:37 Jaimes-MBP package_script_service[96294]: Responsibility set back to self.
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Writing receipt for io.continuum.pkg.main to /Users/jrodriguez
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Writing receipt for io.continuum.pkg.postinstall to /Users/jrodriguez
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Writing receipt for io.continuum.pkg.cacheclean to /Users/jrodriguez
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: No Intel binaries to translate.
Jul 27 20:34:37 Jaimes-MBP installd[96293]: Installed "ExtraFiles" ()
Jul 27 20:34:37 Jaimes-MBP installd[96293]: Failed to add install request history entry: (null)
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: releasing backupd
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: allow user idle system sleep
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Cleared responsibility for install from 6147.
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Cleared permissions on Installer.app
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: ----- End install -----
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: 33.1s elapsed install time
Jul 27 20:34:37 Jaimes-MBP installd[96293]: PackageKit: Removing client PKInstallDaemonClient pid=6147, uid=501 (/System/Library/CoreServices/Installer.app/Contents/MacOS/Installer)
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: PackageKit: Bypassing the enforcement checker since the destination is read/write
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: Running install actions
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: Removing temporary directory "/var/folders/1b/z374sygs1s96t_xxxv0hr3lr0000gn/T//Install.6147K5b4yo"
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: Finalize disk "Macintosh HD"
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: Notifying system of updated components
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: 
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: **** Summary Information ****
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:   Operation      Elapsed time
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: -----------------------------
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:        disk      0.01 seconds
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:      script      0.00 seconds
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:        zero      0.00 seconds
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:     install      34.14 seconds
Jul 27 20:34:38 Jaimes-MBP Installer[6147]:     -total-      34.15 seconds
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: 
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: IFDInstallController 2AB8E0 state = 5
Jul 27 20:34:38 Jaimes-MBP Installer[6147]: Displaying 'Install Succeeded' UI.
```

The thing is that if the tests are unsuccessful, then the installer does fail. Let me push a change to make the tests fail on purpose to confirm.

Reply:
It should only create a file, not visible output… @larsoner

Reply:
Oh I totally missed the redirect `>`, never mind!

Reply:
Hooray `ERROR: executing post_install.sh failed` !